cypress.json

```diff
@@ -6,7 +6,7 @@
   "defaultCommandTimeout": 30000,
   "requestTimeout": 30000,
   "retries": {
-    "runMode": 1,
+    "runMode": 3,
```

Comment:
This will affect all Cypress tests. Are we OK with that? Not sure what the options are if we don't, unless they have a way to specify the config on the command line.

Reply:
I can modify the smoke test commands to specify `retries=3` if we want. i.e. something like 
`npx cypress run -c baseUrl=https://staging.dockstore.org,retries=3 -s my_test.ts`

Reply:
I'll defer to Gary.

Reply:
I actually thought I set it in runMode instead of open. So this is actually what I originally intended.